### ISSUE TITLE
Vac bypass wont work

### ISSUE THREAD

**Issue by azzlak** [danielkrupinski/VAC-Bypass-Loader#113]
i get banned after 30 minutes

**Comment by takoda121** [comment]
🤣😂

**Comment by azzlak** [comment]
i lost my acc

**Comment by takoda121** [comment]
> i lost my acc

duh this isnt working 🤣🤣😂😂

**Comment by Capybaron** [comment]
Are you sure you compiled it properly?

**Comment by azzlak** [comment]
yep

**Comment by msr8** [comment]
Same, got VAC banned within a few days

**Comment by canedoly** [comment]
yeah it doesn't work anymore just got banned, after like 1 month of cheating

**Comment by Capybaron** [comment]
> yeah it doesn't work anymore just got banned, after like 1 month of cheating

Clearly you didn't use it properly

**Comment by msr8** [comment]
Nah, look at all the people suddenly saying it doesnt work, so its much much more probable that it doesn't work against VAC anticheat anymore. And I am dead sure I followed all the steps properly, but it still didnt work

**Comment by Capybaron** [comment]
> Nah, look at all the people suddenly saying it doesnt work, so its much much more probable that it doesn't work against VAC anticheat anymore. And I am dead sure I followed all the steps properly, but it still didnt work

Haven't gotten banned in a year idk what you're on about

**Comment by duwenhao12** [comment]
After cheating on the bypass for a long time, my account suddenly received a ban three days ago.
Maybe steam has new means to fight against bypass loader. After all, steam employees also browse GitHub. So theoretically, any open source cheater and bypass tool will be quickly detected.
The only way to relieve myself of heartache is never to cheat in a high-value master account. I've already suffered

**Comment by Capybaron** [comment]
> After cheating on the bypass for a long time, my account suddenly received a ban three days ago.
> Maybe steam has new means to fight against bypass loader. After all, steam employees also browse GitHub. So theoretically, any open source cheater and bypass tool will be quickly detected.
> The only way to relieve myself of heartache is never to cheat in a high-value master account. I've already suffered

Did you cheat on TF2 or CSGO?

**Comment by moongazer07** [comment]
> > After cheating on the bypass for a long time, my account suddenly received a ban three days ago.
> > Maybe steam has new means to fight against bypass loader. After all, steam employees also browse GitHub. So theoretically, any open source cheater and bypass tool will be quickly detected.
> > The only way to relieve myself of heartache is never to cheat in a high-value master account. I've already suffered
> 
> Did you cheat on TF2 or CSGO?



> After cheating on the bypass for a long time, my account suddenly received a ban three days ago. Maybe steam has new means to fight against bypass loader. After all, steam employees also browse GitHub. So theoretically, any open source cheater and bypass tool will be quickly detected. The only way to relieve myself of heartache is never to cheat in a high-value master account. I've already suffered

i reported some vac exploits to valve, and now they are doing something
